### PR TITLE
STORM-1741: remove unconditional setting of JAVA_HOME from storm-env.sh

### PR DESCRIPTION
**Pull request by ptgoetz**

### CODE REVIEW

File: conf/storm-env.sh [hunk 1]
```diff
@@ -19,6 +19,6 @@
 # Set Storm specific environment variables here.
 
 # The java implementation to use.
-export JAVA_HOME=${JAVA_HOME}
+#export JAVA_HOME=${JAVA_HOME}
```
Comment (erikdw) [inline conf/storm-env.sh]: Maybe leave a comment about why dead code is being left here?

**Comment by ptgoetz** [comment]
As this is a very minor change that fixes a blocker, if there are no objections I will merge it in order to proceed with the 1.0.1 rc3 release.

If there are any objections after the fact, we can cancel the release and revert the change.

**Comment by erikdw** [comment]
Merge away!  Thanks for the update.

**Comment by abhishekagarwal87** [comment]
+1

**Pull request merged**